<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1578</width>
    <height>997</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="horizontalSliderPlano">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Plotter" name="widgetPlotter" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>941</width>
      <height>641</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Plano:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1578</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivos">
    <property name="title">
     <string>Arquivos</string>
    </property>
    <addaction name="actionNovoArquivo"/>
   </widget>
   <widget class="QMenu" name="menuEdi_o">
    <property name="title">
     <string>Edição</string>
    </property>
    <widget class="QMenu" name="menuFormas">
     <property name="title">
      <string>Formas</string>
     </property>
     <addaction name="actionEsfera"/>
     <addaction name="actionElipsoide"/>
     <addaction name="actionVoxel"/>
     <addaction name="actionCaixa"/>
    </widget>
    <addaction name="menuFormas"/>
    <addaction name="actionRGB"/>
   </widget>
   <addaction name="menuArquivos"/>
   <addaction name="menuEdi_o"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNovoArquivo"/>
   <addaction name="separator"/>
   <addaction name="actionRGB"/>
   <addaction name="actionEsfera"/>
   <addaction name="actionElipsoide"/>
   <addaction name="actionVoxel"/>
   <addaction name="actionCaixa"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNovoArquivo">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/recursos/NovoArquivoIMG.png</normaloff>:/icones/recursos/NovoArquivoIMG.png</iconset>
   </property>
   <property name="text">
    <string>NovoArquivo</string>
   </property>
  </action>
  <action name="actionRGB">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/recursos/RGBIMG.png</normaloff>:/icones/recursos/RGBIMG.png</iconset>
   </property>
   <property name="text">
    <string>Cor</string>
   </property>
  </action>
  <action name="actionEsfera">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/recursos/sphere.png</normaloff>:/icones/recursos/sphere.png</iconset>
   </property>
   <property name="text">
    <string>Esfera</string>
   </property>
  </action>
  <action name="actionElipsoide">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/recursos/ellipse.jpg</normaloff>:/icones/recursos/ellipse.jpg</iconset>
   </property>
   <property name="text">
    <string>Elipsoide
</string>
   </property>
  </action>
  <action name="actionVoxel">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/recursos/voxel.png</normaloff>:/icones/recursos/voxel.png</iconset>
   </property>
   <property name="text">
    <string>Voxel</string>
   </property>
  </action>
  <action name="actionCaixa">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/recursos/cubo.png</normaloff>:/icones/recursos/cubo.png</iconset>
   </property>
   <property name="text">
    <string>Caixa</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Plotter</class>
   <extends>QWidget</extends>
   <header location="global">plotter.h</header>
   <container>1</container>
   <slots>
    <signal>moveX(int)</signal>
    <signal>moveY(int)</signal>
    <slot>mudaFrequencia(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSliderPlano</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
